docs/index.md

```diff
@@ -0,0 +1,62 @@
+# web-platform-tests documentation
+
+The web-platform-tests Project is a W3C-coordinated attempt to build a
+cross-browser testsuite for [the Web-platform
```

Comment:
```suggestion
cross-browser test suite for [the Web-platform
```

Reply:
I prefer "test suite," too, but there's precedence beyond this file, so I'd prefer to hold off on this one for now.

Reply:
> I prefer "test suite," too, but there's precedence beyond this file, so I'd prefer to hold off on this one for now.

It’s fine by me to hold of on it for now, but the better solution to deal with the precedence is to globally change “testsuite” anywhere else we have it to “test suite”.

Actually I see now we already have existing instances of “test suite” in the docs, so we have already not been consistent about this.